UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -626,7 +627,11 @@
 
     var singleChoiceForPanels = function (property) {
       var publicId = _.head(property);
-      var propertyValue = (!_.isEmpty(_.last(property))) ? '' : _.last(property);
+      var propertyValue;
+      if (_.last(property) === 0)
+        propertyValue = getProperties(me.pointAsset.newAsset.propertyData,  'additional_panel').defaultValue[publicId];
```

Comment:
Can't this getProperties return  undefined ?

Reply:
no, because this property is defined in the assetTypeConfiguration property data for traffic signs